README.md

```diff
@@ -186,6 +186,135 @@ const App = props => (
 
 Read more : https://github.com/erikras/ducks-modular-redux
 
+## Commit messages
+
+A git repository lives with an history that let developers or automatic procedure to find useful information.
+
+They have to look like that:
+
+```
+type: subject
+
+optional body
+
+footer with references to issue tracker IDS
+```
+
+<details>
+    <summary>See more</summary>
+<p>
+
+##### Type
+
+One of:
+
+- __feat__: a new feature
+- __fix__: a bug fix
+- __docs__: changes to documentation
```

Comment:
Pourquoi mettre un `s` à `docs` et pas à `style` ou `test` ?

Reply:
Je pense que c'est pour être raccord avec commitizen

![https://raw.githubusercontent.com/commitizen/cz-cli/master/meta/screenshots/add-commit.png]

Reply:
That's just what [conventional commits](https://conventionalcommits.org/) says : 

> Others: commit types other than fix: and feat: are allowed, for example commitlint-config-conventional (based on the the Angular convention) recommends chore:, docs:, style:, refactor:, perf:, test:, and others.

Reply:
I don't have an answer as I copied/pasted what udacity does.
But if you want some justification, see https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch that speaks about `/docs`. Every single other prefix are singular.

Reply:
I ask directly here: https://github.com/conventional-changelog/conventionalcommits.org/issues/61